firmware/shared/os/timer.h

```diff
@@ -4,15 +4,16 @@
 #pragma once
 
 #include <cstdint>
-#include "shared/util/os.h"
+
+#include "shared/os/os.h"
 
 namespace shared::os {
 
 class Timer {
```

Comment:
based on discussion, change to `SoftTimer`

Reply:
actually nevermind, this can be the base class for both the software timer and the hardware timer (when we implement it)